demos/handwritten_text_recognition_demo/python/handwritten_text_recognition_demo.py

```diff
@@ -83,10 +83,9 @@ def main():
     net = ie.read_network(args.model, os.path.splitext(args.model)[0] + ".bin")
 
     assert len(net.input_info) == 1, "Demo supports only single input topologies"
-    assert len(net.outputs) == 1, "Demo supports only single output topologies"
 
     input_blob = next(iter(net.input_info))
-    out_blob = next(iter(net.outputs))
+    out_blob = 'output' # name of output blob of model
```

Comment:
I guess these are debug changes. Revert them.

Reply:
It wasn't a debug change, actually. The model uses 5 layers of bi-LSTM so after converting to IR, it has 10 extra LSTM hidden cell outputs as well, and I'm not sure how to discard/suppress those extra outputs. The only two methods I can think of is either calling `next` 10 times on `iter(net.outputs)` or hardcoding the actual output name, as shown in line 88. If there's a better way to handle this, could you please advice? Thanks.
![image](https://user-images.githubusercontent.com/89251456/143367238-ae3c81ea-ad97-47c0-908d-9ba34c93ba9b.png)

Reply:
I've updated `output_blob` to be a command line argument.

Reply:
Can you update 'models/intel/handwritten-english-recognition-0001/README.md' to mention more than 1 output

Reply:
The README has been updated.